spec/configuration_spec.lua

```diff
@@ -125,6 +125,45 @@ describe('Configuration object', function()
       assert.same(services, filter_services(services, { '42' }))
       assert.same({}, filter_services(services, { '21' }))
     end)
+
+    describe("with service filter", function()
+      local mockservices = {
```

Comment:
I think we should also add a test case with a service with more than one host.

For example, we could check that the service is selected when using a regex that does not match the first host, but matches the second or another one.